src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,419 @@
+* **Show appreciation:** ⭐ for helpful responses in support.
+
+Send a message reaction:
+<Code>
+```javascript
+const message; // your message
+await room.messages.reactions.send(message.serial, {name: '✅'});
+```
+</Code>
+
+Message reactions in Ably Chat come in three types: `unique`, `distinct` and `multiple`, to suit different use cases: from one reaction per message to multiple reactions per message with or without counts. See the [Message reactions](/docs/chat/rooms/message-reactions) documentation for more details.
+
+## Message history: essential task context <a id="history"/>
+
+Message history is crucial for task-oriented chats, ensuring continuity and context even when participants join mid-task or return after interruptions.
+
+Ably stores [chat history](/docs/chat/rooms/history) for 30 days by default, with options to extend up to a year on request.
+
+Message history provides users with:
+* **Task continuity:** New participants can quickly understand the current state and previous decisions.
+* **Context preservation:** Users returning to a task don't lose important information.
+* **Audit trail:** Complete conversation records for compliance, training, or dispute resolution.
+
+For task-oriented scenarios, history is almost always beneficial:
+
+* **Support tickets:** Agents can see the full conversation history to understand the issue, after handover from automated/AI support or another agent.
+* **Collaborative work:** Team members can catch up on decisions and progress.
+* **Gaming sessions:** Players can review moves and strategy discussions.
+* **Marketplace transactions:** Complete communication record for orders and deliveries.
+
+<Code>
+```javascript
+// Get the chat room
+const room = await chatClient.rooms.get('support-ticket-123');
+
+// Subscribe to messages
+const subscription = room.messages.subscribe((messageEvent) => {
+  console.log('Received:', messageEvent);
+  // handle message event to update state and UI
+});
+
+// Load recent history for context
+// This ensures you get a complete picture without missing messages
+let historyResponse = await subscription.historyBeforeSubscription({limit: 50});
+
+// Iterate through all pages
+while (true) {
+  console.log('Messages: ', historyResponse.items);
+  if (historyResponse.hasNext()) {
+    historyResponse = await historyResponse.next();
+  } else {
+    break;
+  }
+}
+```
+</Code>
+
+You can use our [React UI Kit](/docs/chat/react-ui-kit) to easily create a [fully featured chat window](/docs/chat/getting-started/react-ui-kit#chat-window) 
+that handles subscribing to messages, loading history, message updates and deletes, message reactions, and more.
+See the [React UI Kit](/docs/chat/react-ui-kit) for more details.
+
+## Enriching tasks with Ably's realtime services
+
+Ably's comprehensive platform enables you to combine chat with other realtime features to create rich, interactive task experiences.
+
+**Pub/Sub channels** add interactive elements:
```

Comment:
Should the wording here be "allow you to build interactive..." so we don't imply these features are out of the box?